packages/expo-audio/src/Audio.types.ts

```diff
@@ -494,24 +499,23 @@ export type AudioMode = {
 };
 
 /**
- * Audio interruption behavior modes for iOS.
+ * Audio interruption behavior modes.
  *
- * Controls how your app's audio interacts with other apps' audio when interruptions occur.
- * This affects what happens when phone calls, notifications, or other apps play audio.
+ * Controls how your app's audio interacts with other apps' audio.
  *
- * @platform ios
+ * - `'doNotMix'`: Requests exclusive audio focus. Other apps will pause their audio.
+ * - `'duckOthers'`: Requests audio focus with ducking. Other apps lower their volume but continue playing.
+ * - `'mixWithOthers'`: Audio plays alongside other apps without interrupting them.
+ *   On Android, this means no audio focus is requested - best suited for sound effects,
+ *   UI feedback, or short audio clips. Note that on Android your app won't receive
+ *   audio focus loss callbacks (e.g., during phone calls) when using this mode.
```

Comment:
```suggestion
 *   audio focus loss callbacks (for example, during phone calls) when using this mode.
```

Let's avoid Latin abbreviations: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations.